### PR TITLE
feat: links to primary listing page on secondary listing page

### PR DESCRIPTION
**Pull request by dontbuyrope**

### CODE REVIEW

File: src/views/collectable/v3/Collectable.vue [hunk 1]
```diff
@@ -188,7 +188,7 @@
             :winningAddress="winningAddress"
             @update-state="updateCollectableState"
           />
-          <div v-if="secondaryMarketListingsForRender?.length > 0" class="text-3xl font-title font-bold text-center mb-6 mt-12" :class="darkMode && 'dark-mode-text'">
```
Comment (dontbuyrope) [inline src/views/collectable/v3/Collectable.vue:191]: aligns the title of this block to the left - like we did for the other items in this list.

File: src/views/collectable/v3/SecondaryCollectable.vue [hunk 2]
```diff
@@ -89,9 +89,16 @@
     </div>
     <container>
       
-      <div class="flex flex-col lg:grid grid-cols-12 gap-12 py-6 pb-32 mt-12 md:mt-0">
+      <div class="flex flex-col lg:grid grid-cols-12 gap-12 py-6 pb-32 mt-6">
         <div class="left-side col-span-7 pb-6">
-          <div class="text-lg description" :class="darkMode ? 'dark-mode-text' : 'text-gray-500'" v-html="description"></div>
+          <sub-title
```
Comment (dontbuyrope) [inline src/views/collectable/v3/SecondaryCollectable.vue:94]: it didn't show the title of the secondary listing - it does now.

File: src/views/collectable/v3/SecondaryCollectable.vue [hunk 3]
```diff
@@ -89,9 +89,16 @@
...
+            class="text-black hidden lg:flex pb-5"
+            textAlign="left"
+          >
+            {{title}}
+          </sub-title>
+          <light-typography class="description" textAlign="left" fontSize="18px">{{description}}</light-typography>
```
Comment (dontbuyrope) [inline src/views/collectable/v3/SecondaryCollectable.vue:100]: copied from the primary listing page.

File: src/views/collectable/v3/SecondaryCollectable.vue [hunk 4]
```diff
@@ -434,6 +457,7 @@ export default {
       console.log({data})
       state.loading = false;
       state.collectable = data.collectable;
+      state.primaryCollectable = data.collectable;
```
Comment (dontbuyrope) [inline src/views/collectable/v3/SecondaryCollectable.vue:460]: on line 463 in this file, the primary is merged with the secondary collectable data, which makes the slug to be overwritten

**Review by JayWelsh — APPROVED** [review]

**Pull request merged**